### PR TITLE
update Lightstep docs with different instructions based on Satellite …

### PR DESCRIPTION
**Pull request by robswhitmore**
…type.

Updated docs on istio.io to reflect installation for different Satellite types

[ ] Configuration Infrastructure
[ x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 1]
```diff
@@ -24,22 +26,26 @@ This task uses the [Bookinfo](/docs/examples/bookinfo/) sample application as an
...
-1.  You'll need to deploy Istio with your satellite address.
-    For on-premise Satellites, ensure you can reach the satellite pool at an address in the format `<Host>:<Port>`, for example `lightstep-satellite.lightstep:9292`.
+## Deploy Istio
+
+How you deploy Istio depends on which type of Satellite you use.
+
+### Deploy Istio with On-Premise Satellites
 
-    For for Public or Developer Satellites, use the address `collector-grpc.lightstep.com:443`.
+These instructions don not assume TLS. If you are using TLS for your Satellite pool, follow the config for the [Public Satellite pool](#deploy-istio-with-public-or-developer-mode-satellites), but
```
Comment (frankbu) [inline content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md:39]:
```suggestion
These instructions do not assume TLS. If you are using TLS for your Satellite pool, follow the config for the [Public Satellite pool](#deploy-istio-with-public-or-developer-mode-satellites), but
```

**Review by frankbu — APPROVED** [review]

**Comment by robswhitmore** [comment]
Hi - is there any way to get this content published to the current release ASAP? I didn't see the instructions for doing that until my branch was already pushed to the main branch. Thanks!

**Pull request merged**